actions.yaml

```diff
@@ -1,4 +1,4 @@
-# Copyright 2021 dwellir
+# Copyright 2024 dwellir
```

Comment:
Why don't this one get to be `Dwellir AB`?

Reply:
Just missed it.